ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx

```diff
@@ -144,6 +144,7 @@ const PostContent = ({ contentBlocks }: { contentBlocks: OptimoBlock[] }) => {
         sizes="(min-width: 1008px) 760px, 100vw"
         className="mediaStyles"
         css={styles.bodyMedia}
+        position={[9]}
```

Comment:
```suggestion
```

Reply:
this is to lazy load images. I could put a real position of a block, but, tbh we want all images lazy loaded as with the banner all images are below the fold